mysql/pt-stalk/trg/pt-stalk-history-list-trigger.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+trg_plugin() {
+  mysql "${EXT_ARGV}" -BNe "SELECT count FROM information_schema.INNODB_METRICS WHERE name = 'trx_rseg_history_len'"|awk '{print $1}';
+}
```

Comment:
It's a good idea to left the last line empty.

Reply:
I'll add the extra space in the last line.